af/fastpath/fastpath/db.py

```diff
@@ -52,14 +53,16 @@ def setup(conf) -> None:
     _ping()
 
 
-def extract_input_domain(msm: dict, test_name: str):
+def extract_input_domain(msm: dict, test_name: str) -> tuple[str, str]:
     """Extract domain and handle special case meek_fronted_requests_test"""
-    input_ = msm.get("input", None)
+    input_ = msm.get("input") or ""
     if test_name == "meek_fronted_requests_test" and isinstance(input_, list):
-        domain = None if input_ is None else urlparse(input_[0]).netloc
-        input_ = ":".join(input_)
+        domain = input_[0]  # type: str
+        input_ = ",".join(input_)
+        input_ = "{" + input_ + "}"
```

Comment:
Is there a reason to change the format of the input for the meek test results?

If we do implement this change, we should test if explorer is able to handle it properly. Otherwise we can maybe stick to the old format of separating the fronts with a `:`?

Reply:
It's what we already have in the database.

Reply:
Oh, I see now that we have in the database a mixture of both formats:
```
SELECT input
FROM fastpath
WHERE (test_name = 'meek_fronted_requests_test') AND (input LIKE '{%')
LIMIT 1

┌─input────────────────────────────────────────────┐
│ {a0.awsstatic.com,d2cly7j4zqgua7.cloudfront.net} │
└──────────────────────────────────────────────────┘
```

and
```
SELECT input
FROM fastpath
WHERE (test_name = 'meek_fronted_requests_test') AND (input NOT LIKE '{%')
LIMIT 1

┌─input──────────────────────────────────────────┐
│ a0.awsstatic.com:d2zfqthxsdq309.cloudfront.net │
└────────────────────────────────────────────────┘
```

Reply:
So I tested the ones prefixed with `{` in explorer and while they don't look very pretty, they don't seem to break.

<img width="1379" alt="Screenshot 2022-03-03 at 18 15 38" src="https://user-images.githubusercontent.com/424620/156616446-caf24f53-fd92-44df-bc45-e07626826228.png">

I guess it's not super high priority to align these to the same format since it's a legacy test.